MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication.swift

```diff
@@ -24,6 +24,28 @@
 
 import Foundation
 
+/**
+    Subclasses MSALPublicClientApplication to be used for Native Auth operations
+    
+    To create an instance of the MSALNativeAuthPublicClientApplication pass directly the clientId, tenantSubdomain, challengeTypes and redirectUri (if needed)
+    to init(clientId: tenantSubdomain: challengeTypes: redirectUri)
+ 
+    For example:
+ 
+     <pre>
+         do {
+             nativeAuth = try MSALNativeAuthPublicClientApplication(
+                 clientId: "Enter_the_Application_Id_Here",
+                 tenantSubdomain: "Enter_the_Tenant_Subdomain_Here",
+                 challengeTypes: [.OOB]
+             )
+             print("Initialised Native Auth successfully.")
```

Comment:
Should we use "MSAL Native Auth" instead?

Reply:
This example was taken from [the documentation](https://github.com/microsoft/entra-previews/blob/PP4/docs/Native-Auth/Developer-guides/1-iOS-Swift/1-Add-authentication-prepare-app.md#create-sdk-instance) so it should stay in sync but can change if needed.